chat.delta.desktop.yml

```
@@ -1,11 +1,11 @@
 app-id: chat.delta.desktop
 base: org.electronjs.Electron2.BaseApp
-base-version: '20.08'
+base-version: '21.08'
 runtime: org.freedesktop.Platform
-runtime-version: '20.08'
+runtime-version: '21.08'
 sdk: org.freedesktop.Sdk
 sdk-extensions:
-  - org.freedesktop.Sdk.Extension.node12
+  - https://github.com/flathub/org.freedesktop.Sdk.Extension.node14
```

This seems to be the problem.

```suggestion
  - org.freedesktop.Sdk.Extension.node14
```